### DISCUSSION TITLE
Compiling 1.19 branch maven dependency help

### DISCUSSION THREAD

**Discussion by arcanebw** [IntelligenceModding/AdvancedPeripherals#354] [Q&A]
Fairly new to the whole gradle setup, Im trying to build a fresh clone with no success, Im simply running `gradlew.bat build` after cloned.  Are some of the repository links outdated?  I would have no idea where to start finding updated links.

A small snippet giving the jist of the error:
```
Could not resolve all files for configuration ':runtimeClasspathCopy'.
> Could not find appeng:appliedenergistics2:12.1.2-beta_mapped_parchment_2022.09.18-1.19.2.
  Searched in the following locations:
    - file:/C:/Users/.../.gradle/caches/forge_gradle/bundeled_repo/appeng/appliedenergistics2/12.1.2-beta_mapped_parchment_2022.09.18-1.19.2/appliedenergistics2-12.1.2-beta_mapped_parchment_2022.09.18-1.19.2.pom
    - file:/C:/Users/.../.gradle/caches/forge_gradle/bundeled_repo/appeng/appliedenergistics2/12.1.2-beta_mapped_parchment_2022.09.18-1.19.2/appliedenergistics2-12.1.2-beta_mapped_parchment_2022.09.18-1.19.2.jar
  Required by:
      project :
> Could not find top.theillusivec4.curios:curio
```

I have no idea what Im doing, and thought this would be the most appropriate place to ask.  I feel as if Im just missing something so trivial to gradle.  I appriciate any help, thank you!

**Comment by SirEndii** [comment]
can you send a full gradle log?
![](https://media.discordapp.net/attachments/572261616956211201/1014887244638732308/unknown.png)

**Reply by arcanebw** [reply]
[gradlewLog.txt](https://github.com/Seniorendi/AdvancedPeripherals/files/9936401/gradlewLog.txt)

Heres the log!  Im not using any IDE, so its just sdout and err redirected to a file.
